project/src/main/utils.gd

```diff
@@ -1,3 +1,4 @@
+tool
```

Comment:
Is this necessary?

Reply:
Yes, otherwise we get errors `Invalid call. Nonexistent function 'assign_card_texture' in base 'Node (utils.gd)'.` We follow the same practice in Turbo Fat.